### ISSUE TITLE
ANSI Escape Sequences Shown in Git Bash

### ISSUE THREAD

**Issue by adamrodger** [Canop/broot#246]
**Describe the bug**
ANSI escape sequences shown in Windows Terminal using Git Bash

**To Reproduce**
Try to run `broot` within Windows Terminal with Git Bash as the shell

**Expected behavior**
The output has proper colours

**Screenshots**
![image](https://user-images.githubusercontent.com/508586/86736029-1e23a280-c02b-11ea-9a0b-8e30b008bf5f.png)

**Configuration (please complete the following information):**
 - OS: Windows
 - Version Windows 10 x64 1904, `broot` 0.18.5
 - Installed via `scoop install broot`

**Comment by Canop** [comment]
It has [already](https://github.com/Canop/broot/issues/98) been reported that broot (like other crossterm based applications) isn't compatible with git bash. I'm sorry but I can't work on that. Other devs who want to tackle that problem are invited to do so (and maybe talk with Timon on https://discord.com/channels/560857607196377088/560857607196377090)

Thanks for the report, though, and please don't delete it, it's a clear one which may help newcomers.

**Comment by adamrodger** [comment]
Yep, no problem. I saw the other issue, but just wanted to raise this one specifically because it's about the ANSI escapes rather than generally being incompatible.

For me, once I've fixed my `~/.bashrc` to `source ~/AppData/Roaming/dystroy/broot/config/launcher/bash/br` then I can get the app to run, but it's unusable because it's just a screen full of ANSI escape sequences.

If the escape sequences can be fixed then I think it would work fine in Git Bash for me. You can make out that it has actually worked within all those escape sequences and formatting problems:

![image](https://user-images.githubusercontent.com/508586/86743478-cee07080-c030-11ea-92cd-a31d762c3519.png)

**Comment by 3tilley** [comment]
Is this still an issue, or has it been fixed with the Windows Terminal improvements?

**Comment by Canop** [comment]
Broot doesn't work in mintty, the terminal that Git for Windows and MSYS2 open by default, as it doesn't provide the Windows console API. Run your Git  Bash or MSYS2 shell in [Windows Terminal](https://github.com/microsoft/terminal) instead: broot works there, including the `br` function.
